src/afl-fuzz.c

```diff
@@ -109,6 +109,7 @@ static void usage(u8 *argv0, int more_help) {
       "maximum.\n"
       "  -m megs       - memory limit for child process (%u MB, 0 = no limit "
       "[default])\n"
+      "  -O            - use binary-only instrumentation (FRIDA mode)\n"
```

Comment:
Again, let's not add a switch? What do you guys think?

Reply:
Dunno, we should be consistent and I don't think we can remove -Q without breaking tons of user scripts built around AFL/AFL++ ...

Reply:
As I mentioned above, I don't think that FRIDA mode can be a binary. I think it needs to be an LD_PRELOADED shared object. So there's a good few little logic changes in the behaviour of AFL which are gated on the `frida_mode` varaible.

Reply:
All right then :) do we have any letters left that are actually part of Frida? Is R taken? Or should we just think of it as the @oleavr mode

Reply:
Here's what's taken...
https://github.com/AFLplusplus/AFLplusplus/blob/687dd9cb6759a0549a04c93d2f073d8cbfba397e/src/afl-fuzz.c#L366